dashboard/config/routes.rb

```diff
@@ -226,7 +226,9 @@
     get 'embed_level', to: 'levels#embed_level', as: 'embed_level'
     post 'update_blocks/:type', to: 'levels#update_blocks', as: 'update_blocks'
     post 'update_properties'
-    post 'clone', to: 'levels#clone'
+    member do
+      post 'clone'
+    end
```

Comment:
This change defines `POST /levels/:id/clone` as an additional member route on the `levels` resource [following Rails conventions](https://guides.rubyonrails.org/routing.html#adding-more-restful-actions), which makes a lot of things work better, particularly CanCanCan authorization helpers.

The behavior without this `member` block is defined in the docs but it's really unclear why you'd want to do it the other way, or what else may not work if you do.  It may be worth going back and moving all of these routes into the `member` block as cleanup.